package.json

```diff
@@ -38,6 +38,7 @@
     "@ls-lint/ls-lint": "^1.11.2",
     "@nuxtjs/eslint-config": "^12.0.0",
     "@nuxtjs/eslint-config-typescript": "^12.0.0",
+    "@nuxtjs/i18n": "^7.3.1",
```

Comment:
do you think we could adequately test this change without adding the module as a dependency?

Reply:
Without the module we can't generate the fixtures to test the "real" problem 😞 
However, I could try something, but not sure if it is the best solution. 
We can utilize both the routes.json files from the fixtures, unsort them, and apply them to new unit tests to evaluate the sortRoute method directly.
Would this be better @danielroe ?

Reply:
Yes, I think that would be fine 👍